src/types/options.ts

```diff
@@ -161,6 +163,10 @@ export interface EventHandlers {
   interactionCreate?: (
     data: Omit<InteractionCommandPayload, "member"> & { member: Member },
   ) => unknown;
+  /** Sent when a user creates a new channel invite. */
+  inviteCreate?: (invite: InviteCreateEvent, channel: Channel) => unknown;
+  /** Sent when an invite is deleted. */
+  inviteDelete?: (invite: InviteDeleteEvent, channel: Channel) => unknown;
```

Comment:
Use the `Camelize` type to convert the keys into camel case and then manually convert the properties into camel case while passing it to the listener.